landing-page/content/common/project/multi-engine-support.md

```diff
@@ -0,0 +1,93 @@
+---
+title: "Multi-Engine Support"
+bookHidden: true
+url: multi-engine-support
+---
+<!--
+ - Licensed to the Apache Software Foundation (ASF) under one or more
+ - contributor license agreements.  See the NOTICE file distributed with
+ - this work for additional information regarding copyright ownership.
+ - The ASF licenses this file to You under the Apache License, Version 2.0
+ - (the "License"); you may not use this file except in compliance with
+ - the License.  You may obtain a copy of the License at
+ -
+ -   http://www.apache.org/licenses/LICENSE-2.0
+ -
+ - Unless required by applicable law or agreed to in writing, software
+ - distributed under the License is distributed on an "AS IS" BASIS,
+ - WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ - See the License for the specific language governing permissions and
+ - limitations under the License.
+ -->
+
+# Multi-Engine Support
+
+Multi-engine support is a core tenant of Apache Iceberg.
+The community continuously improves Iceberg core library components to enable integrations with different compute engines that power analytics, business intelligence, machine learning, etc.
+Support of [Apache Spark](../../../docs/spark-configuration), [Apache Flink](../../../docs/flink) and [Apache Hive](../../../docs/hive) are provided inside the Iceberg main repository.
+
+## Multi-Version Support
+
+Engines maintained within the Iceberg repository have multi-version support.
+This means each new version of an engine that introduces backwards incompatible upgrade has its dedicated integration codebase and release artifacts.
```

Comment:
I think this is only true for Spark and Flink. What about saying that "Iceberg provides a runtime connector Jar for each supported version of Spark and Flink."

We should also note that these are the only additions to the classpath needed. You don't have to add any other dependencies to get support.

Reply:
I see, let me update this section title then and specify this is only for Spark and Flink. I was trying to also make a case for Hive in case it could do that in the future, but it seems unlikely as of today.

Reply:
Updated the doc saying that only Spark and Flink have versioned codebase and jars, Hive as of today continues to use the same one for Hive 2 and 3.